### ISSUE TITLE
Correction in the README

### ISSUE THREAD

**Issue by DhritiShikhar** [sbose78/service-binding-admission-controller#1]
## Question

From https://github.com/sbose78/service-binding-admission-controller/blob/master/README.md#validating-admission-webhook-for-service-binding

> A simple admission webhook which rejects every `ServiceBindingRequest`

Why is the goal of this controller to `reject` Service Binding requests?

## Suggestion

The goal of the controller should be to `validate` Service Binding Requests.

**Comment by sbose78** [comment]
Right, I should have been clearer - this example rejects everything so that it's easy to test this out :)

**Comment by sbose78** [comment]
How does this look https://github.com/sbose78/service-binding-admission-controller/pull/5 ?
